launcher/src/main/java/com/airbnb/mvrx/launcher/MavericksLauncherMockActivity.kt

```diff
@@ -59,6 +59,11 @@ class MavericksLauncherMockActivity : MavericksBaseLauncherActivity() {
          */
         var activityToShowMock: KClass<out Activity> = MavericksLauncherMockActivity::class
 
+        /**
+         * function to call before finishing each mock when running a test all scenario
+         */
+        var onMockLoaded: () -> Unit = {}
```

Comment:
Would recommend changing this lambda to have parameters for the activity, `mock`, and `mockedView`

Reply:
thanks for suggestion, fixed!